import_export/resources.py

```diff
@@ -176,6 +176,12 @@ class ResourceOptions:
     DEFAULT_DB_ALIAS constant ("default") is used.
     """
 
+    store_raw_values = False
+    """
+    If True, The raw data will be stored in the result.
+    Enabling this parameter is going to increase the memory usage and might cause unpredictable results.
```

Comment:
Your English is spot on here :smiley: 

My suggestion would be to modify this slightly:

```
If True, each row's raw data will be stored in each row result.
Enabling this parameter will increase the memory usage during import which should be considered when importing large datasets.
```

Reply:
Thank you.
I fixed your point.